Source/WebCore/WebCore.xcodeproj/project.pbxproj

```diff
@@ -22376,6 +22376,20 @@
 		FFEFAB2D183BCC6F00514567 /* InternalObserverMap.cpp */ = {isa = PBXFileReference; lastKnownFileType = text; path = InternalObserverMap.cpp; sourceTree = "<group>"; };
 /* End PBXFileReference section */
 
+/* Begin PBXFileSystemSynchronizedBuildFileExceptionSet section */
```

Comment:
What's going on with these Xcode project file changes? They look pretty odd for just adding two files.